### PR TITLE
V1.0.3-beta.1 release

### PR DESCRIPTION
**Pull request by sfc-gh-tzhang**
The 1.0.3-beta release has an issue with GCS, so we have to revert https://github.com/snowflakedb/snowflake-ingest-java/pull/334 and then create a new release on top

### CODE REVIEW

File: src/main/java/net/snowflake/ingest/connection/RequestBuilder.java [hunk 1]
```diff
@@ -36,6 +23,16 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.KeyPair;
+import java.util.*;
```
Comment (sfc-gh-rcheng) [inline src/main/java/net/snowflake/ingest/connection/RequestBuilder.java:31]: nit: unsure why these imports were changed?

**Review by sfc-gh-rcheng — APPROVED** [review]
lgtm, one nit, but probably caught by the formatting gate

File: src/main/java/net/snowflake/ingest/connection/RequestBuilder.java (hunk 1 above)
Comment (sfc-gh-japatel) [reply src/main/java/net/snowflake/ingest/connection/RequestBuilder.java:31]: +1 no wildcard import. Probably some intellij magic.

**Review by sfc-gh-japatel — APPROVED** [review]
lgtm

**Review by sfc-gh-tjones — APPROVED** [review]

File: src/main/java/net/snowflake/ingest/connection/RequestBuilder.java (hunk 1 above)
Comment (sfc-gh-tzhang) [reply src/main/java/net/snowflake/ingest/connection/RequestBuilder.java:31]: will update, i update one intellJ so all the settings seems get lost

**Pull request merged**